Changelog.md

```diff
@@ -1,6 +1,7 @@
 ### 0.5.0 (unreleased)
 
 Features:
+ * General: Signed right shift uses proper arithmetic shift, i.e. rounding towards negative infinity.
```

Comment:
For 0.4.0 we had a separate section with breaking changes, I think we should do that again.